### PR TITLE
chore: fix admin version bump

### PR DESCRIPTION
**Pull request by alanthai**
The previous ignore changes is not picking up any `admin` folders.

Also removing the global version bump

### Background

(Provide a link to any relevant issues AND provide a TLDR of the issue in this pull request)

### Solution

(Describe your solution, why this approach was chosen, and what the alternatives/impacts may be)

### 🎩

- ...

### Checklist

- [ ] I have :tophat:'d these changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,8 @@
     "lint": "sk lint",
     "type-check": "sk type-check",
     "build": "sk build",
-    "version-bump": "lerna version --no-push --include-merged-tags",
     "version-bump:checkout": "lerna version --no-push --ignore-changes 'packages/{admin-ui-extensions*,ui-extensions}/**' --include-merged-tags",
-    "version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!({admin*,ui-extensions})/**' --include-merged-tags",
+    "version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!(admin*)/**' --include-merged-tags",
```
[the remark this answers is not in the document]
Comment (alanthai) [reply package.json]: I was thinking after discussion that maybe we want to be more deliberate when version bumping but I think it's not fully fleshed out. I'm going to re-add it (but fixed) until we get a better grasp of the solution.

File: package.json [hunk 2]
```diff
@@ -11,9 +11,8 @@
     "lint": "sk lint",
     "type-check": "sk type-check",
     "build": "sk build",
-    "version-bump": "lerna version --no-push --include-merged-tags",
     "version-bump:checkout": "lerna version --no-push --ignore-changes 'packages/{admin-ui-extensions*,ui-extensions}/**' --include-merged-tags",
-    "version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!({admin*,ui-extensions})/**' --include-merged-tags",
+    "version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!(admin*|ui-extensions)/**' --include-merged-tags",
```
Comment (henrytao-me) [inline package.json:15]: What does `|` do?

Comment (alanthai) [reply package.json:15]: admin* OR ui-extensions. I was just experimenting, and `!(a|b)` seems to work, but the old `!({a,b})` does not.

Comment (alanthai) [reply package.json:15]: the other ones are working with `{}`. I just noticed when I tried version bumping that the combo of `!` and `({})` weren't working for me.

**Review by henrytao-me — APPROVED** [review]
🎩-ed. LGTM

**Pull request merged**